apps/issuer/api.py

```diff
@@ -99,6 +99,59 @@ def put(self, request, **kwargs):
     def delete(self, request, **kwargs):
         return super(IssuerDetail, self).delete(request, **kwargs)
 
+class CollectionBadgeDetail(BaseEntityDetailView):
+    model = CollectionBadgeContainer
+    v1_serializer_class = CollectionBadgeClassSerializerV1
+    # v2_serializer_class = CollectionBadgeClassSerializerV2
+    permission_classes = (AuthenticatedWithVerifiedIdentifier, AuditedModelOwner)
+    valid_scopes = {
+        'get': ['r:issuer', 'rw:issuer'],
+        'post': ['rw:issuer'],
+        'put': ['rw:issuer'],
+        'delete': ['rw:issuer']
+    }
+
+    @apispec_get_operation('CollectionBadge',
+                           summary='Get a single CollectionBadge',
+                           tags=['Issuer']
+                           )
+    def get(self, request, **kwargs):
+        return super(CollectionBadgeDetail, self).get(request, **kwargs)
+
+    @apispec_put_operation('CollectionBadge',
+                           summary='Update a CollectionBadge',
+                           tags=['Issuer']
+                           )
+    def put(self, request, **kwargs):
+        return super(CollectionBadgeDetail, self).put(request, **kwargs)
+
+    @apispec_delete_operation('CollectionBadge',
+                              summary='Delete a collectionBadge',
+                              tags=['Issuer']
+                              )
+    def delete(self, request, **kwargs):
+        return super(CollectionBadgeContainer, self).delete(request, **kwargs)
+
+
+
+class AllSuperBadgeClassesList(UncachedPaginatedViewMixin, BaseEntityListView):
+    """
+    GET a list of superbadgeclasses 
+    """
+    model = SuperBadge
+    v1_serializer_class = SuperBadgeClassSerializerV1
+    v2_serializer_class = SuperBadgeClassSerializerV2
```

Comment:
Is it on purpose that here (in contrast to above) the `v2` isn't commented out?

Reply:
Commenting the ```CollectionBadgeClassSerializerV2``` above was not on purpose, i uncommented it.